package.json

```diff
@@ -26,7 +26,7 @@
     "electron": "1.4.1",
     "request": "2.69.0",
     "infusion": "git://github.com/amb26/infusion.git#240c00be023732ac8dcc193a57bc1b46c58d2de7",
-    "gpii-windows": "git://github.com/gpii/windows.git#hst-2017",
+    "gpii-windows": "git://github.com/kaspermarkus/windows.git#5fc0d9c65db1ac9cff9495f87bc36907d4ecd499",
```

Comment:
This no longer references the latest (and merged) version of the universal GPII-2402 work.... Not sure which branch to best reference here - but I guess @stegru's https://github.com/GPII/windows/pull/122 branch (as soon as he updates the universal reference) would be the most sensible one